arkane/ess/qchem.py

```diff
@@ -264,8 +264,8 @@ def load_conformer(self, symmetry=None, spin_multiplicity=0, optical_isomers=Non
                         rot.append(rotation)
 
                     inertia = []
-
-        modes = mmass + rot + freq
+        # Take only the lat modes found (in the event of multiple jobs)
+        modes = mmass[-1:] + rot[-1:] + freq[-1:]
```

Comment:
Why are these lists in the first place if you only want the last one? I'm skeptical that information that may be wanted in other cases isn't being discarded here.

Reply:
Good question. This and the point about `rot.append(rotation)` makes me think that this code could use a second glance and a cleanup/fixup PR of its own. I'll talk with @alongd about this and open a separate issue.